source/extensions/filters/network/thrift_proxy/router/BUILD

```diff
@@ -13,6 +13,7 @@ envoy_cc_extension(
     name = "config",
     srcs = ["config.cc"],
     hdrs = ["config.h"],
+    category = "SOMECAT",
```

Comment:
see https://github.com/envoyproxy/envoy/pull/14744/files#r560952318

Reply:
k - so i guess `envoy.thrift_proxy.transports` here

Reply:
looking in `extensions_build_config.bzl` this is marked as `envoy.filters.thrift.router` but im not seeing a factory in the source code for that

Reply:
hmm, so im back to thinking this sholuld be `envoy.thrift_proxy.transports`

Reply:
I would think filters, but @zuercher can weight in on this definitively.

Reply:
ill mark this as `"envoy.thrift_proxy.filters` for now - we can change if we need to or as becomes apparent